pkgs/tools/security/otpgen/default.nix

```diff
@@ -34,11 +35,18 @@ stdenv.mkDerivation rec {
       url = "https://github.com/magiruuvelvet/OTPGen/commit/afd3fc87c4422bdf4f3978d4a2de1565a1d93283.patch";
       sha256 = "1ib96dkb4szvmlf0d4n92gxngfxg05lncibxsndqynjq7sl50n4q";
     })
+
   ];
 
   nativeBuildInputs = [ cmake pkg-config ];
 
-  buildInputs = [ cryptopp qtbase ];
+  buildInputs = [ cryptopp zlib qtbase ];
+
+  preConfigurePhases = [ "prePatch" ];
```

Comment:
This should be unnecessary, `prePatch` will be automatically run by `patchPhase`.

Reply:
Oh, interesting, I didn't know prePatch existed.

Reply:
Yeah, almost every phase runs `pre` and `post` hook these days: https://nixos.org/nixpkgs/manual/#ssec-patch-phase

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/6de88bc942dad407746fc6642b8ff3eb31077323